### DISCUSSION TITLE
Any plans / way to sync with razer chroma?

### DISCUSSION THREAD

**Discussion by chapsterrulez** [ixjf/MSIRGB#191] [Q&A]
I have an MSI B450 A Pro Max that doesn't natively support syncing with razer chroma connect. Any way you can add this or would there be something I could experiment with?
Cheers

**Comment by ixjf** [comment]
MSIRGB was never meant as a one-stop tool for configuring RGB. I have no plans to support anything other than MSI motherboards like mine. I couldn't either way since I don't have any of that hardware to test my code on.

You can check OpenRGB, see if it does what you want.
